theme/views.py

```diff
@@ -220,6 +220,22 @@ def signup(request, template="accounts/account_signup.html", extra_context=None)
     return HttpResponseRedirect(request.META['HTTP_REFERER'])
 
 
+def signup_verify(request, uidb36=None, token=None):
+    """
+    Signup verify. Overriding mezzanine's view function for signup verify
+    """
+    user = authenticate(uidb36=uidb36, token=token, is_active=False)
+    if user is not None:
+        user.is_active = True
+        user.save()
+        auth_login(request, user)
+        info(request, _("Successfully signed up"))
+        return HttpResponseRedirect('/user/{}/?edit=true'.format(user.id))
```

Comment:
@dtarb wanted the signup verify to redirect to the user profile in edit mode.  I had to override the mezzanine signup_verify method in order to do that because mezzanine uses login_redirect.  There is also a js change to check for an edit query param and set the profile to edit mode if provided.  Perhaps @Maurier could take a moment to look it over and see if what I did was the right way to do it.